### PR TITLE
Update quickbooks from 2019 to 2020

### PR DESCRIPTION
**Pull request by suschizu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by suschizu** [comment]
A Xcode10.1 travis problem:
`An error occurred while installing nokogiri (1.10.4), and Bundler`

File: Casks/quickbooks.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'quickbooks' do
-  version '2019'
+  version '2020'
   sha256 :no_check # required as upstream package is updated in-place
 
   url "https://http-download.intuit.com/http.intuit/Downloads/#{version}/Latest/QuickBooksMac#{version}.dmg"
-  appcast 'https://http-download.intuit.com/http.intuit/Downloads/Mac/QuickBooks/018/9DBF6268-EEBF-431D-9E50-DE3FEAE52FA2/appcast.xml'
+  appcast 'https://http-download.intuit.com/http.intuit/Downloads/Mac/QuickBooks/019/8D4DADF6-42C2-4D64-ABC0-D28253C30D19/appcast.xml'
```
Comment (reitermarkus) [inline Casks/quickbooks.rb]: This should be part of the version.

Comment (reitermarkus) [reply Casks/quickbooks.rb]: Ah, I thought this was the URL, nevermind.

Comment (suschizu) [reply Casks/quickbooks.rb]: but the appcast was wrong too :)

**Pull request merged**